enacts/monthly/maproom.py

```diff
@@ -170,25 +163,26 @@ def update_map(variable, month):
     )
     def pick_location(click_lat_lng):
         if click_lat_lng == None:
-            rr_mrg = read_data("precip")
+            rr_mrg = calc.read_enacts_zarr_data(**READ_PARAMS)
             return [
                 ((rr_mrg["Y"][int(rr_mrg["Y"].size/2)].values)),
                 ((rr_mrg["X"][int(rr_mrg["X"].size/2)].values)),
             ]
-        return click_lat_lng                           #  in the data to where the user clicked on the map.
+        return click_lat_lng # in the data to where the user clicked on the map.
```

Comment:
I don't understand this comment (the "in the data" part). Do you? Seems like it might have been the second line of a two-line comment, and the first line got deleted.

Reply:
done